lms/static/js/learner_dashboard/views/entitlement_unenrollment_view.js

```diff
@@ -52,11 +52,7 @@
                     this.resetModal();
                     this.setHeaderText(courseName, courseNumber);
                     this.setSubmitData(apiEndpoint);
-
-                    if (window.edx && window.edx.dashboard && window.edx.dashboard.dropdown) {
-                        window.edx.dashboard.dropdown.toggleCourseActionsDropdownMenu(event);
-                        this.$el.css('position', 'fixed');
-                    }
```

Comment:
This code hides the gear dropdown menu when the unenroll link is clicked, which is problematic for a11y as it prevents us from returning focus to the previously focused element when the modal is closed.